packages/testing/src/api/index.ts

```diff
@@ -109,15 +109,10 @@ function initializeApp(
 ): firebase.app.App {
   let appOptions = {};
   if (databaseName) {
-    appOptions = {
-      databaseURL: `http://${DATABASE_ADDRESS}?ns=${databaseName}`
-    };
-  } else if (projectId) {
-    appOptions = {
-      projectId: projectId
-    };
-  } else {
-    throw new Error('neither databaseName or projectId were specified');
```

Comment:
we still need this check don't we?

Reply:
The underlying `FirebaseApp` already checks that `databaseURL` is defined when `.database()` is called, and that `projectId` is defined when `.firestore()` is called. I think these are just as intuitive as our existing error message.